### PR TITLE
FIX ISSUE :#6975 Remove all references to ArenaV1

### PR DESCRIPTION
**Pull request by chakra2017**
Description: Remove all references to ArenaV1 FIX-#6975

- [x] Delete the record of the file nginx.rewritesnamed: arenav1.
- [x] Delete the file /arena/index.phpthat will no longer be used.
- [x] Delete the controller function Contestcalled: getContestListDetailsForTypeScript.
- [x] Delete the record of the file webpack.config-frontend.jsnamed: arena_contest_list.
- [x] Delete the file /frontend/www/js/omegaup/arena/contest_list.tsthat will no longer be used.
- [x] Delete the component /components/arena/ContestList.vuethat will no longer be used.
- [x] Delete all tests that call anything from what was previously deleted.

### CODE REVIEW

**Comment by anmode** [comment]
Please run ` ./stuff/lint.sh fix -- frontend/server/src/Controllers/Contest.php` and the push. This will solve the lint issue.

File: frontend/server/src/Controllers/Contest.php [hunk 1]
```diff
@@ -1050,117 +1050,7 @@ public static function getContestPracticeDetailsForTypeScript(
...
-                    'countContests' => $countContestsByType,
-                ],
-                'title' => new \OmegaUp\TranslationString(
-                    'omegaupTitleContestList'
-                ),
-            ],
-            'entrypoint' => 'arena_contest_list',
-        ];
-    }
+
```
Comment (pabo99) [inline frontend/server/src/Controllers/Contest.php:1053]: Tihs line could be removed too
```suggestion
```

Comment (chakra2017) [reply frontend/server/src/Controllers/Contest.php:1053]: Ok .. will do thank you

**Comment by pabo99** [comment]
One more thing: Could  you run:
```
./stuff/lint.sh
```
outside the container please? It should fix the linter errors

**Comment by pabo99** [comment]
In order to solve the lint error, you should run:
```
./stuff/lint.sh fix -- frontend/server/src/Controllers/Contest.php
```

**Comment by pabo99** [comment]
This PR is now redundant as the issue has been resolved in PR #7007

**Pull request closed without merging**